snow/consensus/snowball/consensus_performance_test.go

```diff
@@ -28,12 +28,12 @@ func TestSnowballOptimized(t *testing.T) {
 
 	sampler.Seed(seed)
 	for i := 0; i < numNodes; i++ {
-		nBitwise.AddNode(&Tree{})
+		nBitwise.AddNode(NewTree)
```

Comment:
I'm wondering if it's possible for us to pass parameters into `NewTree` and `NewFlat` so we can avoid passing in this function directly